### PR TITLE
Fix openapi jackson bug

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

FLA-1250

Fix swagger
update corresponding code

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/libs/efiling-bambora-api-client/pom.xml [hunk 1]
```diff
@@ -141,7 +141,7 @@
 			<plugin>
 				<groupId>org.openapitools</groupId>
 				<artifactId>openapi-generator-maven-plugin</artifactId>
-				<version>4.3.1</version>
+				<version>5.1.1</version>
```
Comment (akroon3r) [inline src/backend/libs/efiling-bambora-api-client/pom.xml:144]: bold.

File: src/backend/libs/efiling-bambora-api-client/src/main/java/ca/bc/gov/open/jag/bambora/PaymentConstants.java [hunk 2]
```diff
@@ -10,8 +10,8 @@ private  PaymentConstants() {
 
     }
     public static final String COURT_SERVICES = "COURT SERVICES";
-    public static final int CARD_ID = 1;
-    public static final int BAMBORA_APPROVAL_RESPONSE = 1;
+    public static final Integer CARD_ID = 1;
+    public static final String BAMBORA_APPROVAL_RESPONSE = "1";
```
Comment (akroon3r) [inline src/backend/libs/efiling-bambora-api-client/src/main/java/ca/bc/gov/open/jag/bambora/PaymentConstants.java:14]: bolder.

Comment (TayGov) [reply src/backend/libs/efiling-bambora-api-client/src/main/java/ca/bc/gov/open/jag/bambora/PaymentConstants.java:14]: ☕

File: src/backend/libs/efiling-bambora-api-client/pom.xml (hunk 1 above)
Comment (TayGov) [reply src/backend/libs/efiling-bambora-api-client/pom.xml:144]: ☕

**Comment by TayGov** [comment]
> very bold.

☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕  ☕ ☕

**Pull request merged**